kyaml/openapi/openapi.go

```diff
@@ -72,16 +73,15 @@ func AddSchemaFromFile(path string) error {
 
 // DeleteSchemaInFile reads the file at path and removes all the openAPI definitions
 // present in file from global schema
-func DeleteSchemaInFile(openAPIPath string) error {
+func DeleteSchemaInFile(openAPIPath string) {
```

Comment:
Would it be better if instead of having a separate `DeleteSchemaInFile` function, the `AddSchemaFromFile` function would return a cleanup function? Presumably that would remove the need to read the same file twice might make the use of `defer` clearer.

Reply:
That's a good idea. Will do that.